scripts/create.sh

```diff
@@ -22,14 +22,14 @@ set -e
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 # Include the user set variables
-# shellcheck source=properties.env
+# shellcheck source=properties.env disable=SC1091
```

Comment:
`# shellcheck disable=SC1091`

Please

Reply:
You have it in multiple places, can we touch properties.env or run shell check after terraform??

Reply:
```
# shellcheck source=properties.env

In ./scripts/validate.sh line 27:
source "${ROOT}/properties.env"
^-- SC1090: Can't follow non-constant source. Use a directive to specify location.
```
```
# shellcheck disable=SC1091

In ./scripts/validate.sh line 24:
source "${ROOT}/properties.env"
^-- SC1091: Not following: properties.env: openBinaryFile: does not exist (No such file or directory)
```

And both included makes a clean lint run:
```
# shellcheck source=properties.env disable=SC1091

Running shellcheck
Running flake8
Running go fmt and go vet
Running terraform validate
Running hadolint on Dockerfiles
Checking for required files
Checking file headers
Verifying boilerplate in file: Makefile as Makefile
Verifying boilerplate in file: test/make.sh as sh
Verifying boilerplate in file: test/verify_boilerplate.py as py
Verifying boilerplate in file: scripts/validate.sh as sh
Verifying boilerplate in file: scripts/teardown.sh as sh
Verifying boilerplate in file: scripts/create.sh as sh
Verifying boilerplate in file: scripts/update-db-ratings.sh as sh
All files examined have correct boilerplate.
The following lines have trailing whitespace
```

To your question: We can either exclude them from shellcheck and make no changes to the local repo or add logic to the `test/make.sh` to touch empty files to satisfy shellcheck but potentially alter the state of the repo.  I was opting for the former as the safest option.